1558753219790	Sync.LogManager	DEBUG	Flushing file log
1558753219791	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1558753219791	FirefoxAccounts	DEBUG	getOAuthToken enter
1558753219795	FirefoxAccounts	TRACE	getCachedToken returning cached token
1558753219795	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1558753219795	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1558753219795	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1558753219795	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1558753219795	Services.Common.RESTRequest	TRACE	HTTP Header if-none-match: "4841a94e0eb8f5b17f2d54bd6293df8865f63d2b-gzip"
1558753219805	Services.Common.RESTRequest	TRACE	onStartRequest: GET https://profile.accounts.firefox.com/v1/profile
1558753219805	Services.Common.RESTRequest	TRACE	Channel for GET https://profile.accounts.firefox.com/v1/profile returned status code 2152398878
1558753219805	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1558753219805	Services.Common.RESTRequest	TRACE	GET body: 
1558753219806	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]"}
1558753219806	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]"}
1558753219819	Sync.LogManager	DEBUG	Log cleanup threshold time: 1557889219819
1558753219909	Sync.LogManager	DEBUG	Done deleting files.
1558753220745	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558753221889	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558753232193	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558753462831	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1558753462831	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558753462833	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558753463830	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558753464580	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1558753464580	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1558753464588	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Linux x86_64) FxSync/1.69.0.20190521202118.desktop
1558753464588	Sync.Service	INFO	Starting sync at 2019-05-24 21:04:24 in browser session Z_DfCFv3mZcd
1558753464588	Sync.Service	DEBUG	In sync: should login.
1558753464588	Sync.Service	INFO	User logged in successfully - verifying login.
1558753464592	Sync.BrowserIDManager	DEBUG	unlockAndVerifyAuthState already has (or can fetch) sync keys
1558753464594	Sync.Status	DEBUG	Status.login: error.login.reason.network => success.status_ok
1558753464594	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1558753464594	Sync.Service	DEBUG	Fetching unlocked auth state returned success.status_ok
1558753464597	Sync.BrowserIDManager	INFO	Getting keys
1558753464598	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1558753464598	FirefoxAccounts	DEBUG	enter getAssertion()
1558753464603	FirefoxAccounts	DEBUG	got keyPair
1558753464603	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1558753464609	Services.Common.RESTRequest	TRACE	local time, offset: 1558753464609, 0
1558753464615	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1558753464615	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1558753464615	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1558753464615	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: es-mx,en-us,en
1558753464616	Services.Common.RESTRequest	DEBUG	POST Length: 888
1558753464616	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"2DDEE3B3B40A355B9278FA81A0DC8EC826BCB3DBACBB00ED723CF6D9A288D2366D16F60F6EF82C26185E58B93C2FDE8B28729749357733532EF146625D9F515894FBC9A378DD6439AC595FAD430F1221D49F4AB659EC8BDDDFA26BA9AC8FC80D91EF255B896536EF5C13E3EE06380B2EF52DDA95040BDF76D70FB76A1F4CC8B6","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1558753464621	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1558753464621	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398878
1558753464622	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1558753464622	Services.Common.RESTRequest	TRACE	POST body: 
1558753464622	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:434
1558753464623	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
request()@resource://services-common/hawkclient.js:224
1558753464624	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
request()@resource://services-common/hawkclient.js:227
1558753464624	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1558753464624	Services.Common.RESTResponse	TRACE	Processing response headers.
1558753464624	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:234
1558753464624	Services.Common.RESTResponse	TRACE	Processing response headers.
1558753464625	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:235
1558753464625	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
_constructError()@resource://services-common/hawkclient.js:116
request()@resource://services-common/hawkclient.js:240
1558753464625	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:117
request()@resource://services-common/hawkclient.js:240
1558753464626	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:118
request()@resource://services-common/hawkclient.js:240
1558753464626	Services.Common.RESTResponse	TRACE	Processing response headers.
1558753464626	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:240
1558753464626	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1558753464626	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1558753464627	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1558753464627	Sync.Status	DEBUG	Status.login: success.status_ok => error.login.reason.network
1558753464627	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1558753464627	Sync.BrowserIDManager	INFO	Failed to fetch the cluster URL: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1558753464628	Sync.Service	DEBUG	verifyLogin failed: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1558753464628	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1558753464628	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1558753464628	Sync.ErrorHandler	ERROR	Sync encountered a login error
1558753464628	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558753464629	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558753464632	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.network(resource://services-sync/service.js:869:15) JS Stack trace: onNotify@service.js:869:15
1558753464633	Sync.Service	DEBUG	Not syncing: login returned false.
